packages/manager/apps/web/client/app/hosting/multisite/translations/Messages_fr_FR.json

```diff
@@ -8,6 +8,14 @@
   "hosting_multisite_statistics_period_week": "Semaine",
   "hosting_multisite_statistics_period_month": "Mois",
   "hosting_multisite_statistics_period_year": "Année",
+  "hosting_multisite_dialog_title": "Verification des serveurs DNS",
+  "hosting_multisite_dialog_close": "Fermer",
+  "hosting_multisite_dialog_modify_a_record": "Modifiez le champ de type A avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} geré sur les serveurs DNS {{dns}}",
+  "hosting_multisite_dialog_modify_a_record_same_nic": "Modifier le champ de type A",
+  "hosting_multisite_dialog_modify_a_record_external_nic": "Modifiez le champ de type A avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} geré sur les serveurs DNS {{dns}} (la modification doit être effectuée par l'administrateur de la zone DNS: {{nic}}",
+  "hosting_multisite_dialog_modify_a_record_error": "Ajoutez un champ de type A en renseignant l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} géré sur les serveurs DNS {{dns}}",
+  "hosting_multisite_dialog_modify_a_record_same_nic_error": "Modifier le champ de type A",
+  "hosting_multisite_dialog_modify_a_record_external_nic_error": "Modifiez le champ de type A avec l'adresse IP {{ip}} dans la zone DNS du domaine {{domain}} geré sur les serveurs DNS {{dns}} (la modification doit être effectuée par l'administrateur de la zone DNS: {{nic}}",
```

Comment:
Is it normal that content is the same that "hosting_multisite_dialog_modify_a_record_external_nic" ?

Reply:
yes